src/index.html

```diff
@@ -18,5 +18,66 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="aimg"
+        href="dfrf"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <a
+          class="nav__link is_active"
+          href="1"
+        >
+          Apple
+        </a>
+        <a
+          class="nav__link"
+          href="2"
+        >
+          Samsung
+        </a>
+        <a
+          class="nav__link"
+          href="3"
+        >
+          Smartpfones
```

Comment:
The text 'Smartpfones' seems to be a typo. If it's meant to be 'Smartphones', please correct it.